### PR TITLE
Resolving event refresh after event is deleted

### LINKED ISSUE #2259: Deleted events stays in cache until refreshed
**Describe the bug**

-Whenever you delete an event,A tooltip appears showing event is deleted.
-Event stays in cache until the page is refreshed.
**To Reproduce**

1.)Go to '/Events/Live'
2.)Try to delete an event.
3.)Refresh the page .
4.)See error

**Expected behavior**

When a event is deleted , The page should be refreshed to remove the event from the list.
**Screenshots**

kushprince

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Chrome
 - Version: 75.0.0.2

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Whenever an event is deleted from 'Admin/Events' Or from 'Events/Live' , It stays in cache until the page is refreshed.

#### Changes proposed in this pull request:

-Page reloads after any event is deleted or restored.

Fixes #2259

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 Please review for any errors in it.

File: app/controllers/events/list.js [hunk 1]
```diff
@@ -73,13 +73,13 @@ export default Controller.extend({
...
       })
         .then(() => {
+          this.set('isLoading', false);
           this.notify.success(this.get('l10n').t('Event has been deleted successfully.'));
+          this.send('refreshRoute');
         })
         .catch(() => {
-          this.notify.error(this.get('l10n').t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
```
Comment (CosmicCoder96) [inline app/controllers/events/list.js]: why did you remove the `finally` block.

Comment (kushthedude) [reply app/controllers/events/list.js]: Since the due to the  'finally' block 'An unexpected block' was getting executed again ,So when you will delete an event 'Event has been deleted successfully' And 'An unexpected error' Both of them gets appear on tooltip. So to avoid that i removed the finally handler of promise.

Comment (CosmicCoder96) [reply app/controllers/events/list.js]: finally block needs to be intact, and should be responsible for setting `isLoading` to `false`. `isLoading` does not need to be set `false` separately for success and failure cases.

Comment (kushthedude) [reply app/controllers/events/list.js]: I did the changes , Looks fine now?

**Comment by kushthedude** [comment]
@CosmicCoder96 review?

**Comment by CosmicCoder96** [comment]
I tried this multiple times, I am unable to reproduce the issue. Events disappear as soon as they are deleted.

**Comment by kushthedude** [comment]
@CosmicCoder96 try to delete event from ‘’manage events” section and if you still cant reproduce the issue then i will close it becuse on my machine it was persistent before this change😅

**Comment by CosmicCoder96** [comment]
> @CosmicCoder96 try to delete event from ‘’manage events” section and if you still cant reproduce the issue then i will close it becuse on my machine it was persistent before this change

Yeah I did, can someone else confirm this @Anupam-dagar @uds5501 ?

**Comment by uds5501** [comment]
@kushthedude No, the issue doesn't occur neither on admin route nor on manage event route. This PR can be closed :)

**Comment by kushthedude** [comment]
@uds5501 can you tell me the possible cause of this issue on my local machine?

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 @kushthedude I can't reproduce this issue too.

**Comment by kushthedude** [comment]
@Anupam-dagar i think it was something related to my browser issue and yup i closed this issue😅. Will take care next time before opening new issue.
